keras_cv/models/generative/stable_diffusion/stable_diffusion.py

```diff
@@ -283,6 +283,144 @@ def generate_image(
         decoded = ((decoded + 1) / 2) * 255
         return np.clip(decoded, 0, 255).astype("uint8")
 
+    def inpaint(
+        self,
+        prompt,
+        image,
+        mask,
+        num_resamples=1,
+        batch_size=1,
+        num_steps=25,
+        unconditional_guidance_scale=7.5,
+        diffusion_noise=None,
+        seed=None,
+    ):
+        """Generates an image based on encoded text.
+
+        The encoding passed to this method should be derived from
```

Comment:
it looks like the prompt is encoded internally in this method.

Lets update the docstring to match